client/components/plans/plan-price/index.jsx

```diff
@@ -23,8 +23,9 @@ const PlanPrice = React.createClass( {
 				return this.translate( 'Free', { context: 'Zero cost product price' } );
 			}
 
-			const monthlyPrice = +( rawPrice / 12 ).toFixed( 2 );
-			formattedPrice = formattedPrice.replace( rawPrice, monthlyPrice );
+			const currencySymbol = formattedPrice.slice( 0, 1 );
```

Comment:
are we sure that the currency symbol will always be first? The Euro sign in many parts of Europe and the Israeli Shekel come to mind as ones that come after the value.